src/api/spec/controllers/attribute_controller_spec.rb

```diff
@@ -0,0 +1,50 @@
+require 'rails_helper'
+
+RSpec.describe AttributeController, type: :controller do
+  render_views
+
+  describe '#update' do
+    let(:admin) { create(:admin_user) }
+    let(:user) { create(:confirmed_user) }
+    let(:attribute) { build(:attrib_type) }
+    let(:namespace) { attribute.namespace }
+    let(:name) { attribute.name }
```

Comment:
:thought_balloon: why do this indirection for `name` and `namespace`?

Reply:
to save typing?